CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.16.0 [2023-04-11]
+
+_Dependencies_
+
+- Recompiled plugin with steampipe-plugin-sdk v5.3.0 which adds go-getter support to dynamic tables. ([#340](https://github.com/turbot/steampipe-plugin-alicloud/pull/340))
```

Comment:
@madhushreeray30 could you please add the changelog link to the steampipe-plugin-sdk v5.3.0 version?

Reply:
@misraved  ok